Fix:#91 Improve introductory section

smitrajput opened the pull request:
<!-- Describe what your PR does -->
Improve the 'What is this' section to include a meaningful crux of the core features, the CLI offers.
Note: It has a dummy-logo, which is a mere placeholder/suggestion to the protofire team
## Cute animal picture
:poodle: 
<!-- Include a picture of a cute animal. This is not mandatory, but the reviewers will be very sad if you don't do it. -->

fvictorio commented:
Thanks @smitrajput, this looks good! Don't worry about the CI, that's caused by the format of the commit messages. When this is ready to be merged I'm going to squash it and change the message.

@pablofullana could you also take a look?

ChrisChinchilla commented:
@smitrajput You don't really need to use HTML in markdown, you especially don't need to use heading tags, or paragraph tags, better to use markdown syntax instead.

ChrisChinchilla commented on README.md:
```diff
@@ -58,6 +65,19 @@ section or the [list of commands](#commands) to find out what you can do with it
 
+## Installation
+
+Install it globally:
```
```suggestion
Install es-cli globally:
```

fvictorio replied:
Just in case, it's `eth-cli`

ChrisChinchilla commented on README.md:
```diff
@@ -58,6 +65,19 @@ section or the [list of commands](#commands) to find out what you can do with it
+
+`npm install -g eth-cli`
+
+You can also try it out with `npx`:
```
```suggestion
You can also try es-cli out with `npx`:
```

ChrisChinchilla commented:
Hey @smitrajput I've been working with these bounties to help polish before merging, just a couple of small things and we can get it merged and paid out.

smitrajput commented:
> @smitrajput You don't really need to use HTML in markdown, you especially don't need to use heading tags, or paragraph tags, better to use markdown syntax instead.

I've got rid of the h1 and the 'p' tags, but I had to use the <center> tag instead, is that OK? Coz this is the only way I found other than html's, to center align texts. Or if you have a better alternative in mind to center align texts, plz let me know

ChrisChinchilla commented:
@smitrajput Why does it need to be centred? This is just a read me file, most just use the default markdown syntax.

smitrajput commented:
> @smitrajput Why does it need to be centred? This is just a read me file, most just use the default markdown syntax.

@ChrisChinchilla , just to make it look better. If the left alignment of the heading 'eth-cli' is better for you, then I think I should keep it left-aligned. Opinions?

ChrisChinchilla commented:
@smitrajput I'd say just stick to vanilla markdown, GitHub readmes are typically pretty plain, and the content is the most important. :)

smitrajput commented:
> @smitrajput I'd say just stick to vanilla markdown, GitHub readmes are typically pretty plain, and the content is the most important. :)

Made the following suggested changes:
- removed all HTML tags
- removed the logo, as it looked a bit odd, with the left-aligned texts
- replaced 'it' with 'eth-cli' wherever suggested
Please have a look

fvictorio commented:
@smitrajput When everything is ready, please squash everything into a single commit with a message like `chore: improve readme introduction`. I can do it for you, but in that case I will appear as the committer, and I prefer that credit is properly given :slightly_smiling_face:

smitrajput commented:
@ChrisChinchilla please have a look at my last commit

ChrisChinchilla commented on README.md:
```diff
@@ -58,6 +59,19 @@ section or the [list of commands](#commands) to find out what you can do with it
+
+Install eth-cli globally:
+
+`npm install -g eth-cli`
```
```suggestion
```shell
npm install -g eth-cli
```
```

ChrisChinchilla commented:
@smitrajput One tiny suggestion and we're good to go @fvictorio

smitrajput commented:
@ChrisChinchilla please have a look now

fvictorio commented:
@smitrajput can you squash everything into a single commit with a message like `chore: improve intro in readme`?

smitrajput commented:
> @smitrajput can you squash everything into a single commit with a message like `chore: improve intro in readme`?

@fvictorio I think I have squashed it already into a single commit. You can verify this by checking out the 'Commits' tab in this pull request, which shows '1 commit', namely '9aa1ef1'

fvictorio commented:
Yes, that worked. Thanks @smitrajput!

smitrajput commented:
@fvictorio I haven't yet received the bounty :sweat_smile:

ChrisChinchilla commented:
@smitrajput @fvictorio doesn't have it, we will ow process it, but they are in an America time zone.

smitrajput commented:
> @smitrajput @fvictorio doesn't have it, we will ow process it, but they are in an America time zone.

Received. Thanks!
